tests/test_light.py

```diff
@@ -2084,12 +2084,12 @@ async def test_light_state_restoration(
         xy_color=None,
         hs_color=None,
         color_mode=None,
-        effect=None,  # Effect is the only `None` value actually restored
```

Comment:
The workaround of restoring `effect` even if `None` is no longer needed/present, so the tests are adjusted.